python/pyspark/sql/tests/streaming/test_streaming_foreach_family.py

```diff
@@ -0,0 +1,369 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import tempfile
+import time
+
+from pyspark.testing.sqlutils import ReusedSQLTestCase
+
+
+class StreamingTestsForeachFamilyMixin:
```

Comment:
Better not to have Mixin class yet. We don't have corresponding '_partity_' tests in connect for this.

Reply:
Sure I'll remove the mixin. Still better to create this class though, to avoid tons of `@unittest.skip` in the streaming parity test class.

Reply:
Class is fine.